From d676fe7d3284c953c72e07231df3e8990ce89bea Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:25:18 -0400
Subject: [tools] perf: Ignore 'perf timechart' output file

Message-id: <1408462094-14194-150-git-send-email-jolsa@redhat.com>
Patchwork-id: 88162
O-Subject: [PATCH RHEL7.1 BZ1131394 149/325] perf tools: Ignore 'perf timechart' output file
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit f3c236b0c7a84e5d59cc639a1673a20b0a59ecc0
Author: Ramkumar Ramachandra <artagnon@gmail.com>
Date: Thu Oct 3 14:10:36 2013 +0530

description
===========
The default output file produced by the 'perf timechart' tool is called
output.svg, add it to .gitignore.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/.gitignore b/tools/perf/.gitignore
index 8f8fbc2..782d86e 100644
--- a/tools/perf/.gitignore
+++ b/tools/perf/.gitignore
@@ -13,6 +13,7 @@ perf*.html
 common-cmds.h
 perf.data
 perf.data.old
+output.svg
 perf-archive
 tags
 TAGS
-- 
1.7.1